release-1.8/release_notes_draft.md

```diff
@@ -250,6 +271,18 @@ kind.
 #### kube-proxy ipvs mode
 * [alpha] Support ipvs mode for kube-proxy([#46580](https://github.com/kubernetes/kubernetes/pull/46580), [@haibinxie](https://github.com/haibinxie))
 
+### API Machinery
+
+* [alpha] The CustomResourceDefinition API can now optionally perform validation
+  on custom objects based on a JSON Schema provided in the CRD spec.
+  This alpha feature must be enabled with the `CustomResourceValidation` feature gate
```

Comment:
Enable this alpha feature with the `CustomResourceValidation` feature gate in `kube-apiserver`.